src/renderer/components/dock/logs/call-for-all-logs.injectable.ts

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
This is unnecessary indirection to `callForLogs`. It shouldn't be coupled with `Pod`.

Replace usage of this with: `callForLogs({ name: pod.getName(), namespace:  pod.getNs() }, query)`